src/generic-container/generic-container.ts

```diff
@@ -185,6 +186,11 @@ export class GenericContainer implements TestContainer {
     return this;
   }
 
+  public withVolumes(volumes: string | string[]): this {
```

Comment:
Need to add this method to the base interface `TestContainer` (https://github.com/testcontainers/testcontainers-node/blob/91d9127f3e157a59f7371bec59effc5889096b1d/src/test-container.ts#L19)

Reply:
done